gridfs/__init__.py

```diff
@@ -861,8 +880,9 @@ def open_download_stream_by_name(self, filename, revision=-1, session=None):
             raise NoFile(
                 "no version %d for filename %r" % (revision, filename))
 
-    def download_to_stream_by_name(self, filename, destination, revision=-1,
-                                   session=None):
+    def download_to_stream_by_name(self, filename: str, destination: Any,
+                                   revision: Optional[int] = -1,
+                                   session: Optional[ClientSession] = None) -> None:
```

Comment:
I saw elsewhere you used NoReturn. Should we change all the `-> None:` methods to use it too?

Reply:
Those are for functions that always raise an exception